### ISSUE TITLE
Make `attr` sort meaningful?

### ISSUE THREAD

**Issue by mhkeller** [hhsnopek/pourover#7]
Since you seem to have taken this project over, per https://github.com/NYTimes/pourover/issues/42, what do you think about the feasibility of making sorts by attribute name along and easily make ascending / descending versions?

**Comment by hhsnopek** [comment]
Yeah lets definitely get that working. If you got time to PR it in, that would be great!

**Comment by mhkeller** [comment]
I took a look last year but it's a bit beyond me unfortunately.

**Comment by hhsnopek** [comment]
I'll take a look at it later tonight to see where it can be added in

**Comment by hhsnopek** [comment]
I'm going to add this in v2 unless someone else wants to take a whack at it
